molecule/source-static/molecule.yml

```diff
@@ -22,24 +22,26 @@ lint: |
   capabilities:
     - NET_ADMIN
     - NET_RAW
+  pre_build_image: true
```

Comment:
If the images are being pre_built, then where are they hosted? I do not see them on dockerhub or quay.io.

I also do not see any cronjob task to automatically build them, although that could be done later.

Reply:
I manually built and pushed them,
my idea is to automate it in a future PR
they are at pulp dockerhub:
https://hub.docker.com/repository/docker/pulp/molecule_fedora31
https://hub.docker.com/repository/docker/pulp/molecule_centos8
https://hub.docker.com/repository/docker/pulp/molecule_debian10